python/docs/source/index.md

```diff
@@ -3,94 +3,119 @@ hide-toc: true
-    <h1 class="display-5 fw-bold">Event-processing for AI applications.</h1>
+    <h1 class="display-5 fw-bold">Real-Time AI without the fuss.</h1>
     <div class="col-lg-7 mx-auto">
-      <p class="lead mb-4">Next-generation, real-time and historic event processing.
+      <p class="lead mb-4">Kaskada is a simple streaming engine that connects AI models to real-time & historical data.
       </p>
     </div>
 </div>
 
+(what-it-provides)=
+## Kaskada completes the Real-Time AI stack, providing...
+
 ```{gallery-grid}
 :grid-columns: 1 2 2 3
 
-- header: "{fas}`timeline;pst-color-primary` Real-time processing for all"
-  content: "Quickly process events so you can respond in real-time."
-  link: ".#stream"
-- header: "{fab}`python;pst-color-primary` Python-native"
-  content: "Use Python so you can load data, process it, and train and serve models from one place."
-  link: ".#python"
-- header: "{fas}`gauge-high;pst-color-primary` Get started immediately"
-  content: "No infrastructure to provision let's you jump right in."
-  link: ".#get-started"
-
-- header: "{fas}`fast-forward;pst-color-primary` Real-time, Batch and Streaming"
-  content: "Execute large-historic queries or materialize in real-time. Or both."
-  link: ".#real-time-and-historic"
-- header: "{fas}`rocket;pst-color-primary` Local, Remote and Distributed"
-  content: "Develop and test locally. Deploy to Docker, K8s or a service for production."
-  link: ".#local-and-distributed"
-- header: "{fas}`backward;pst-color-primary` Time-travel"
-  content: "Generate training examples from the past to predict the future."
-  link: ".#time-travel"
+- header: "{fas}`timeline;pst-color-primary` Stream Processing"
+  content: "Precompute model inputs from streaming data with robust data connectors, transformations & aggregations."
+- header: "{fas}`binoculars;pst-color-primary` Event Detection"
+  content: "Trigger pro-active application behavior by identifying meaningful changes in your data as they happen."
+- header: "{fas}`backward;pst-color-primary` History Replay"
+  content: "Backtest and fine-tune from historical data using per-example time travel and point-in-time joins."
 ```
 
-* * *
-
-(stream)=
-# Real-time event-processing
-
-Kaskada is built on Apache Arrow, providing an efficient, columnar representation of data.
-The same approach is at the core of many analytic databases as well as Pandas and Polars.
 
-Kaskada goes beyond the columnar representation, by introduce a Timestream -- a columnar representation of events, ordered by time and grouped by key.
-This representation is a perfect fit for all kinds of events, modern event streams as well as events stored in a database.
-Specializing for Timestreams allows Kaskada to optimize temporal queries and execute them much faster.
+(real-time-ai)=
+### Real-time AI in minutes
 
-(python)=
-# Python-native
+Connect and compute from streaming data _or_ databases, and dynamically load data using Python.
```

Comment:
Connect and compute over databases, streaming data, _and_ data loaded dynamically using Python.